src/managers/KeybindManager.cpp

```diff
@@ -720,7 +720,13 @@ void CKeybindManager::centerWindow(std::string args) {
 
     const auto PMONITOR = g_pCompositor->getMonitorFromID(PWINDOW->m_iMonitorID);
 
-    PWINDOW->m_vRealPosition = PMONITOR->vecPosition + PMONITOR->vecSize / 2.f - PWINDOW->m_vRealSize.goalv() / 2.f;
+    auto       RESERVEDOFFSET = Vector2D();
+    try {
+        if (std::stoi(args) == 1)
```

Comment:
is this necessary? Can't it just be `if (args == "1")`?

Reply:
lol